hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/TarContainerPacker.java

```diff
@@ -110,8 +112,8 @@ public byte[] unpackContainerData(Container<KeyValueContainerData> container,
               StandardCopyOption.ATOMIC_MOVE,
               StandardCopyOption.REPLACE_EXISTING);
     } else {
-      throw new IOException("Unpack destination directory " + destContainerDir
-              + " is not empty.");
+      throw new StorageContainerException("Container unpack failed because " +
+          "ContainerFile already exists", CONTAINER_ALREADY_EXISTS);
```

Comment:
Please add container ID or destContainerDir in the error message for a easier diagnose.